packages/ffe-form-react/src/InputGroup.js

```diff
@@ -66,14 +86,25 @@ class InputGroup extends Component {
                 ? children(extraProps)
                 : React.cloneElement(children, extraProps);
 
+        const error =
+            typeof fieldMessage === 'string' ? (
+                <ErrorFieldMessage element="p">
+                    {fieldMessage}
+                </ErrorFieldMessage>
+            ) : (
+                React.isValidElement(fieldMessage) && fieldMessage
+            );
+
         return (
             <div
                 className={classNames(
                     'ffe-input-group',
                     { 'ffe-input-group--no-extra-margin': !extraMargin },
-                    { 'ffe-input-group--error': !!fieldMessage },
+                    { 'ffe-input-group--error': shouldShowError },
                     className,
                 )}
+                onChange={this.onChange}
+                onBlur={this.onBlur}
```

Comment:
Because of this change, the linter throws an error saying "Static HTML elements with event handlers require a role  jsx-a11y/no-static-element-interactions".
Adding `onChange` and `onBlur` to a `div` generally feels a bit.. weird - but it works! But I don't know what aria role to give this div.

Should I
a) Give the div some role?
b) Ignore the linter rule
c) Pass the `onBlur` and `onChange` down in `extraProps` doing something along the lines of
```
const extraProps = {
  ......,
  onBlur: () => {
    this.onBlur();
    if(this.props.onBlur) {
      this.props.onBlur()
    }
  }
}
```
Solution c) might risk the `onBlur` and `onChange` never being called if they're never applied in the children.